README.md

```diff
@@ -114,6 +130,6 @@ Your Answer:
 Select all that apply:
 ```
 [] Synchronous code runs at an even pace, asynchronous code runs with uneven pacing.
-[] Synchronous code runs all at the same time, asynchronous code runs completely randomly
+[X] Synchronous code runs all at the same time, asynchronous code runs completely randomly
```

Comment:
Synchronous code runs in order (as appears in the source).  That means one line runs, then the other.  They are not running "at the same time".
Asynchronous code may run at a later time. Such as when a button is clicked.